### ISSUE TITLE
Cannot open jpg, gif, or png in code-server web IDE

### ISSUE THREAD

**Issue by bowdens07** [cdr/code-server#3936]
## OS/Web Information

- Web Browser: chrome/firefox/edge
- Local OS:  Microsoft Windows 10 Pro 10.0.19042 Build 19042
- Remote OS: N/A
- Remote Architecture: N/A
- `code-server --version`: 3.11.1 and 3.11.0

## Steps to Reproduce

1. Open a docker image of code-server with the specified versions above
2. Upload a jpg, gif, or png image into the web IDE 
3. Double click the uploaded file to display the image

## Expected

The image should display in a webview without error

## Actual

Same error message below appears in chrome, firefox, and edge

![cannotopenpng](https://user-images.githubusercontent.com/79893142/128759953-c7aa7248-532a-4a30-baf4-5fbfb26b8dd2.PNG)

## Notes

There was a regression between versions 3.10.2 and 3.11.0. The bug does not occur on versions earlier than 3.11.0. There is no output on the console whenever this error occurs, which is 100% of the time. I am going to attempt to isolate the version which the bug starts. I will comment when I have done this. 

This issue can be reproduced in VS Code: No

**Comment by jsjoeio** [comment]
Thanks for the thorough report @bowdens07 - I think you're right about 3.11.0 introducing this regression. In my console, I see:
```js
Refused to load the image 'https://vscode-remote+localhost.vscode-resource.vscode-webview.net:8080/Users/jp/dev/folder-for-testing/download.png?version%3D1628621135878' because it violates the following Content Security Policy directive: "img-src data: https://*.vscode-webview.net".
```

## Video 

https://user-images.githubusercontent.com/3806031/128917587-d993edc9-8c41-4928-b28c-70b0e80e1be7.mov

If I had to make a guess, I bet it's this PR that introduced it: https://github.com/cdr/code-server/pull/3895

I'm not sure why "it violates the following Content Security Policy directive" 🤔 cc @code-asher

**Comment by code-asher** [comment]
Ahh maybe the CSP no longer works because the URL has a port in it now?
So to fix I think we can patch the directive to be:

```
img-src data: https://*.vscode-webview.net:*
```

**Comment by jsjoeio** [comment]
This is probably in the webview directory. Once that upstream patch lands in VS Code and we update, we won't have to worry about this.

**Comment by galesky** [comment]
For the record, the issue on Duckly was that we were importing a Script on runtime (like a script tag on the extension webview)

This has been changed by using webpack to pre-bundle all scripts a single HTML using [InlineChunkHtmlPlugin](https://github.com/facebook/create-react-app/blob/bb64e31a81eb12d688c14713dce812143688750a/packages/react-dev-utils/InlineChunkHtmlPlugin.js)

**Comment by jsjoeio** [comment]
Taking a look here...

Going to use this comment as a place to take notes.

This seems to be the solution that might work but unsure where to put it 🤔 

```text
img-src data: https://*.vscode-webview.net:*
```

Doing a search for "Content-Security-Policy" to find a couple places. In the meantime, going to run `yarn watch` and reproduce it locally.

**Comment by kongdd** [comment]
I have the same issue with the lastest code-server, v4.5.1

**Comment by jsjoeio** [comment]
can you open a new issue?
